packages/dom/src/test/dom.js

```diff
@@ -200,6 +201,70 @@ describe( 'DOM', () => {
 			expect( () => stripHTML( html ) ).not.toThrow();
 		} );
 	} );
+
+	describe( 'offsetIframe', () => {
+		it( 'returns rect without changes if element is not an iframe', () => {
+			const rect = {
+				left: 50,
+				top: 50,
+				bottom: 100,
+				right: 100,
+				width: 50,
+				height: 50,
+			};
+			const offsettedRect = offsetIframe( rect, parent.ownerDocument );
+
+			expect( offsettedRect ).toEqual( rect );
+		} );
+
+		it( 'returns offsetted rect if element is in an iframe', () => {
+			const iframeLeft = 25;
+			const iframeTop = 50;
+			const childLeft = 10;
+			const childTop = 100;
+
+			const iframe = document.createElement( 'iframe' );
+			parent.appendChild( iframe );
+			// JSDom doesn't have a layout engine
+			// so we need to mock getBoundingClientRect and DOMRect.
+			iframe.getBoundingClientRect = () => ( {
```

Comment:
To make it clearer that we're mocking this function, and to fall more in line with idiomatic jest, could we use a jest method like `mockImplementation`? (not sure if that is the exact one)

```javascript
iframe.getBoundingClientRect = jest.mockImplementation(() => ( {
				width: 100,
				height: 100,
				top: iframeTop,
				left: iframeLeft,
			} ));
```

Reply:
Great idea! Wrapped them in `jest.fn( impl )` which is basically a shorthand for `jest.fn().mockImplementation( impl )`.